.tekton/update-dependencies.yaml

```diff
@@ -0,0 +1,180 @@
+            value: "refs/heads/$(params.source_branch)"
+      - name: update-dockerfiles
+        runAfter:
+          - git-clone
+        workspaces:
+          - name: source
+            workspace: source
+        params:
+          - name: repo_name
+            value: $(params.repo_name)
+          - name: repo_owner
+            value: $(params.repo_owner)
+          - name: source_branch
+            value: $(params.source_branch)
+          - name: target_branch
+            value: "robot/$(params.source_branch)/update_dockerfiles"
+        taskSpec:
+          params:
+            - name: repo_name
+            - name: repo_owner
+            - name: source_branch
+            - name: target_branch
+          results:
+            - name: skip-open-pr
+              description: Flag that controls whether or not to open a PR
+              type: string
+          steps:
+            - name: setup-local-repository
+              image: quay.io/redhat-pipeline-service/dependencies-update:main
+              imagePullPolicy: Always
+              script: |
+                #!/usr/bin/env bash
+                set -o errexit
+                set -o pipefail
+                set -x
+                cd "$(workspaces.source.path)"
+
+                # Setup user
+                git config --local user.email "pipeline-service@example.com"
+                git config --local user.name "Pipeline Service CI Robot"
+
+                # Use SSH authentication
+                git config --replace-all remote.origin.url "$(
+                  git config --get remote.origin.url | \
+                    sed -e "s|^https\?://github.com/|git@github.com:|" -e "s|\(\.git\)\?$|.git|"
+                )"
+
+                # Set the branch
+                git checkout -b "$(params.source_branch)"
+            - name: update-dockerfiles
+              image: quay.io/redhat-pipeline-service/dependencies-update:main
+              imagePullPolicy: Always
+              script: |
+                #!/usr/bin/env bash
+                set -o errexit
+                set -o pipefail
+                set -x
+                cd "$(workspaces.source.path)"
+
+                /opt/dependencies-update/bin/update.sh \
+                  --commit_to "$(params.target_branch)" \
+                  --task update_dockerfiles_base_images_sha \
+                  --workspace_dir "$(workspaces.source.path)"
+            - name: is-pr-required
+              image: quay.io/redhat-pipeline-service/dependencies-update:main
+              imagePullPolicy: Always
+              env:
+                - name: GITHUB_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: github
+                      key: token
+              script: |
+                #!/usr/bin/env bash
+                set -o errexit
+                set -o pipefail
+                set -x
+                echo OK
+                cd "$(workspaces.source.path)"
```

Comment:
You could specify the property `workingDir="(workspaces.source.path)"` at this step, then you can remove this line.  for example
```
- name: is-pr-required
   image: quay.io/redhat-pipeline-service/dependencies-update:main
   imagePullPolicy: Always
   workingDir="(workspaces.source.path)"
```

Reply:
Done